manim/mobject/graphing/coordinate_systems.py

```diff
@@ -1744,6 +1668,25 @@ def construct(self):
         Whether or not to include the tips on both axes.
     kwargs : Any
```

Comment:
```suggestion
    kwargs
```
Port over docs typehints to in-code ones?

Reply:
Hmm, only `kwargs` were missed.
We haven't type hinted them anywhere, do you still want me to?

Reply:
Ah I see, then my comment is invalid. Plus, I don't think kwargs should be typehinted.